### PR TITLE
feature/225 add Schedule C1 to Data Dictionary

### PR DESCRIPTION
**Pull request by dheitzer**
contains proposed solution to use the field prefixes (H, G, etc) in the title and field description only.

### CODE REVIEW

File: schema/SchC1.json [hunk 1]
```diff
@@ -0,0 +1,1019 @@
...
+                "COL_SEQ": 28,
+                "FIELD_DESCRIPTION": "E.5 IND/NAME (Account Location)",
+                "TYPE": "A/N-200",
+                "REQUIRED": null,
+                "SAMPLE_DATA": null,
+                "VALUE_REFERENCE": null,
+                "RULE_REFERENCE": null
+            }
+        },
+        "street_1": {
```
Comment (toddlees) [inline schema/SchC1.json:603]: I would prefix E.6-10 with "account_"
like "account_street_1"

**Review by toddlees — DISMISSED** [review]

File: schema/SchC1.json [hunk 2]
```diff
@@ -0,0 +1,1019 @@
...
+                "COL_SEQ": 26,
+                "FIELD_DESCRIPTION": "E.3 ESTIMATED VALUE",
+                "TYPE": "AMT-12",
+                "REQUIRED": null,
+                "SAMPLE_DATA": null,
+                "VALUE_REFERENCE": null,
+                "RULE_REFERENCE": null
+            }
+        },
+        "date_depository_account_established": {
```
Comment (mjtravers) [inline schema/SchC1.json:564]: In keeping with our variable pattern for most date fields (date_signed and date_of_election look to be the exceptions), suggest we rename this to depository_account_established_date

**Review by mjtravers — DISMISSED** [review]
~~The COL SEQ values are missing from the SchC1_spec.html file. The column header is there but the values for FIELD DESCRIPTION are in the COL SEQ column.~~
Comment OBE: The generate-spec-table.py script was updated and the missing COL SEQ fixed

File: schema/SchC1.json [hunk 3]
```diff
@@ -0,0 +1,1019 @@
...
+        "authorized_date_signed"
+    ],
+    "fec_recommended": [
+        "lender_street_1",
+        "lender_city",
+        "lender_state",
+        "lender_zip",
+        "loan_interest_rate"
+    ],
+    "properties": {
```
Comment (mjtravers) [inline schema/SchC1.json:32]: fec_recommended property is OBE and can be removed

File: schema/SchC1.json (hunk 2 above)
Comment (mjtravers) [reply schema/SchC1.json]: change was made

File: schema/SchC1.json (hunk 1 above)
Comment (mjtravers) [reply schema/SchC1.json]: change was made

**Review by mjtravers — APPROVED** [review]

**Pull request merged**